opentelemetry-stdout/src/metrics/exporter.rs

```diff
@@ -76,57 +76,139 @@ impl PushMetricsExporter for MetricsExporter {
     }
 
     fn shutdown(&self) -> Result<()> {
-        self.writer.lock()?.take();
+        self.is_shutdown.store(true, atomic::Ordering::SeqCst);
         Ok(())
     }
 }
 
+fn print_metrics(metrics: &[ScopeMetrics]) {
+    for (i, metric) in metrics.iter().enumerate() {
+        println!("\tInstrumentation Scope #{}", i);
+        println!("\t\tName         : {}", &metric.scope.name);
+        if let Some(version) = &metric.scope.version {
+            println!("\t\tVersion  : {:?}", version);
+        }
+        if let Some(schema_url) = &metric.scope.schema_url {
+            println!("\t\tSchemaUrl: {:?}", schema_url);
+        }
+        metric
+            .scope
+            .attributes
+            .iter()
+            .enumerate()
+            .for_each(|(index, kv)| {
+                if index == 0 {
+                    println!("\t\tScope Attributes:");
+                }
+                println!("\t\t\t ->  {}: {}", kv.key, kv.value);
+            });
+
+        metric.metrics.iter().enumerate().for_each(|(i, metric)| {
+            println!("Metric #{}", i);
+            println!("\t\tName         : {}", &metric.name);
+            println!("\t\tDescription  : {}", &metric.description);
+            println!("\t\tUnit         : {}", &metric.unit);
+
+            let data = metric.data.as_any();
+            if let Some(hist) = data.downcast_ref::<data::Histogram<i64>>() {
```

Comment:
We only allow `i64` for UpDownCounter and Gauge, right? [Histograms](https://github.com/open-telemetry/opentelemetry-rust/blob/main/opentelemetry-sdk/src/metrics/meter.rs#L376-L399) should only be tested for `u64` and `f64`? 
```suggestion
            if let Some(hist) = data.downcast_ref::<data::Histogram<u64>>() {
```